readthedocs/doc_builder/exceptions.py

```diff
@@ -66,6 +66,15 @@ class BuildUserError(BuildBaseException):
         "Add a configuration file to your project to make it build successfully. "
         "Read more at https://docs.readthedocs.io/en/stable/config-file/v2.html"
     )
+    BUILD_IMAGE_CONFIG_KEY_DEPRECATED = gettext_noop(
+        'The configuration key "build.image" is deprecated. '
+        'Please, use "build.os" instead to make it build successfully. '
```

Comment:
A couple copy changes:

- We can be more authoritative without "please"
- The "it" is ambiguous
- Avoid "successfully" as a successful build is up to the maintainer

```suggestion
        'Use "build.os" instead to continue building your project. '
```

Reply:
Loved your suggestion with the explanation of each word 💯